cmd/vulcan-prowler/main.go

```diff
@@ -360,6 +418,8 @@ func parseControl(raw string) (control string, description string, err error) {
 	// parts[0] = [check13 .
 	control = strings.Replace(parts[0], "[check", "", -1)
 	// control = 13 .
+	control = control[:1] + "." + control[1:]
```

Comment:
One last thing, if `control` is empty, this command will triggers a `slice bounds out of range` error. Can we put a protection/check for that?

Reply:
I think it is a good suggestion,  the prowler tool must always return a non empty string, but it is true that it is better to check for that precondition.